doc/build.rst

```diff
@@ -316,7 +316,8 @@ like ``-DCMAKE_CXX_COMPILER``.
 
 .. warning::
 
-   MSVC is not supported for the R package as it has difficulty handling R C headers.
+   MSVC is not supported for the R package as it has difficulty handling R C
+   headers. CMake build is not supported either.
```

Comment:
Perhaps this warning could be put higher up.

Reply:
Done 

![image](https://github.com/user-attachments/assets/720fcdb5-fd9c-44f9-88fc-0e202a66c66b)